float64 timestamp
anchor[10] anchors
estimate estimate
uint8 nAnchors
